firebase-firestore/src/commonMain/kotlin/dev/gitlive/firebase/firestore/Timestamp.kt

```diff
@@ -27,6 +27,7 @@ expect class Timestamp internal constructor(nativeValue: NativeTimestamp): BaseT
     object ServerTimestamp: BaseTimestamp
 }
 
-fun Timestamp.Companion.fromMilliseconds(milliseconds: Double): Timestamp =
-    Timestamp((milliseconds / 1000).toLong(), (milliseconds * 1000).toInt() % 1000000)
-fun Timestamp.toMilliseconds(): Double = seconds * 1000 + (nanoseconds / 1000.0)
+private const val FACTOR = 1000000
```

Comment:
well, 1 ms is not 1000 nanos, but 1000000

Reply:
It makes sense to add a comment for the "FACTOR" constant. and if you have already extracted the factor for nanos, why don't you do it for miles?